app/controllers/idv/enter_password_controller.rb

```diff
@@ -1,14 +1,13 @@
 module Idv
   class EnterPasswordController < ApplicationController
-    before_action :personal_key_confirmed
-
     include IdvStepConcern
     include StepIndicatorConcern
     include PhoneConfirmation
```

Comment:
Noticed PhoneConfirmation here. I don't think it's used in this controller. In fact it looks like the one method it defines isn't used in several of the places it's included. Maybe that's a different PR...